lib/puppet/type.rb

```diff
@@ -1019,8 +1018,7 @@ def insync?(is)
 
     if property = @parameters[:ensure]
       unless is.include? property
-        raise Puppet::DevError,
-          "The is value is not in the is array for '#{property.name}'"
+        raise Puppet::DevError, _("The 'is' value is not in the 'is' array for '%{name}'") % { name: property.name }
```

Comment:
added quotes around 'is'

Reply:
Should this have a translation note that `is` is a literal thing, and shouldn't be translated?